engine/runtime/src/main/java/org/enso/interpreter/runtime/error/PanicSentinel.java

```diff
@@ -12,10 +16,10 @@
  * <p>This tracing is enabled by the active intervention of the runtime instrumentation, and does
  * not function in textual mode.
  */
+@ExportLibrary(value = InteropLibrary.class, delegateTo = "panic")
 @ExportLibrary(MethodDispatchLibrary.class)
-public class PanicSentinel extends RuntimeException implements TruffleException {
-  private final PanicException panic;
-  private final Node location;
+public class PanicSentinel extends AbstractTruffleException {
```

Comment:
This should not be an `AbstractTruffleException` nor export the exceptions library. Notice how this makes the panic sentinels handlable in language, which is exactly opposite to what we want.